Repo-Integration/Self-Managed/setup.sh

```diff
@@ -96,16 +97,33 @@ echo "${grn}Download Success!!!${end}"
 }
 
 function key_check(){
-## Look for Activation Key
-if [ -z "${ws_key}" ]
+## Look for Activation Key and github.com token
+if [ -z "${ws_key}" ] && [ -z "${WS_KEY}" ]
```

Comment:
why check for different case on this variable, but not for the github_com_token?

Reply:
@samq-ws because env vars are always supposed to be uppercase, but I wanted to keep ws_key lowercase working since that's how it used to work.  I could do it for GITHUB too if you think it's good

Reply:
I think you misunderstood my comment.  See line [120](https://github.com/mend-toolkit/mend-examples/blob/5eb859d3d1c90ac4d781a6515534fcaebb8c43a3/Repo-Integration/Self-Managed/setup.sh#L120)

Reply:
Yes @samq-ws line 113 looks for both uppercase and lowercase variables.  If neither is set, we error out.  If only the lower-case is set, then line 122 sets the uppercase one to be the value of the lowercase one.  Same logic as for ws_key.